05-Shared_computing_etiquette.Rmd

```diff
@@ -14,17 +14,17 @@ ottrpal::include_slide("https://docs.google.com/presentation/d/1B4LwuvgA6aUopOHE
+First, let's learn a bit about this JHPCE. For this particular resource, there are about 400 active users. It is optimized for genomic and biomedical research and has 4,000 cores! That's right: as you can imagine, this is much more powerful than the individual laptops and desktops that researchers at the university have for personal use, which would typically only have around 8 cores. The JHPCE also has 28TB of RAM and 14 PB of storage!
 
 Now that you know more about digital sizes, you can appreciate that this server can allow for much faster processing and really large amounts of storage, as again a researchers' computer might have something like 16 GB of RAM and 1TB of storage. 
 
-There are 68 nodes that make up the JHPCE currently. As, with most clusters some of the nodes are dedicated to managing users logging in to the cluster and some of the nodes are dedicated to data transferring. Each node has 2-4 CPUs that provide 24-128 cores! As you can see these processors or chips have a lot more cores per each CPU than a typical personal computer. 
+Currently, there are 68 nodes that make up the JHPCE currently. As with most clusters, some of the nodes are dedicated to managing user access to the cluster, and some of the nodes are dedicated to transferring data. Each node has 2-4 CPUs that provide 24-128 cores! As you can see, these processors or chips have a lot more cores per each CPU than a typical personal computer. 
 
-Individual users connect and perform jobs (aka computational tasks) on the cluster using a formal [common pool resource (CPR)](https://en.wikipedia.org/wiki/Common-pool_resource) hierarchy system. What does this mean? This means that it is a shared resource, where if one user overused the resource it would be to the detriment of others and to overcome this there are usage rules and regulations that are enforced by managers of the resource @common-pool_2022.  This is important because if a single or a few users used up all the computing resources one day, then the other nearly 400 users would have to delay their work that day, which would not be fair. 
+Individual users connect and perform jobs (a.k.a. computational tasks) on the cluster using a formal [common pool resource (CPR)](https://en.wikipedia.org/wiki/Common-pool_resource) hierarchy system. What does this mean? Recall that we are talking about shared resources, where if one user overused the resource, it may be to the detriment of other users' experiences. The CPR hierarchy system is comprised of rules and regulations designed to prevent this from happening, that are enforced by managers of the resource @common-pool_2022. This is important because if a single or a few users used up all the computing resources one day, then the other nearly 400 users would not be able to perform their work that day, which would not be fair. 
```

Comment:
```suggestion
Individual users connect and perform jobs (a.k.a. computational tasks) on the cluster using a formal [common pool resource (CPR)](https://en.wikipedia.org/wiki/Common-pool_resource) hierarchy system. What does this mean? Recall that we are talking about shared resources, where if one user overuses the resource, it would be to the detriment of other users' experiences. Rules and regulations are designed to prevent this from happening and are enforced by managers of the resource @common-pool_2022. This is important because if a single or a few users used up all the computing resources one day, then the other nearly 400 users would not be able to perform their work that day, which would not be fair. 
```